app/commands/v2/discard_draft.rb

```diff
@@ -47,7 +47,7 @@ def delete_draft_from_draft_content_store(draft_path)
       def send_live_to_draft_content_store(live)
         PresentedContentStoreWorker.perform_async(
           content_store: Adapters::DraftContentStore,
-          payload: Presenters::ContentStorePresenter.present(live, event),
+          payload: Presenters::ContentStorePresenter.present(live, event, [:draft, :published]),
```

Comment:
Yeah this is the fallback order: there is a comment copied here:

We need to support an array of states to cater for the DiscardDraft
command which deletes the draft content item and sends the published item
to the draft content store. This means that we need to try to find a
draft, but fall back to the published item (if it exists).